.github/workflows/tests.yml

```diff
@@ -67,12 +67,12 @@ jobs:
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
       - name: Test Types
-        run: yarn run test:types --colors
+        run: yarn run test:types --target 4.9,5.0,latest
```

Comment:
If I got it right, TypeScript 4.9 is currently lowest supported version. Pick any (;

It would be fun to setup a cron job with `tstyche --target next` to test on nightly releases.